extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -5,6 +5,7 @@
 
 import * as path from 'path';
 import * as vscode from 'vscode';
+import { homedir } from 'os';
```

Comment:
import breaks the web build, needs to update fallback in https://github.com/microsoft/vscode/blob/6896254c1240bab3e7ab3c9028039f3fd2c2b4c9/extensions/shared.webpack.config.js#L118

```
    Module not found: Error: Can't resolve 'os' in '/mnt/vss/_work/1/s/extensions/typescript-language-features/src'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "os": require.resolve("os-browserify/browser") }'
	- install 'os-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "os": false }
Details:
    domainEmitter: [object Object]
    domainThrown: false

```

Reply:
Will check, I had a feeling this would break